packages/cw0/src/macros.rs

```diff
@@ -0,0 +1,34 @@
+/// Quick check for a guard. If the condition (first argument) is false,
```

Comment:
Is this a good place for this macro? If we'd commit to using it, basically every contract could use it so some `packages/utils` seems more appropriate imo.

Reply:
I think it belongs in cosmwasm_std. But once it is there it is hard to change.

Putting it here temporarily so we can experiment with it.

Reply:
That said, renaming cw0 to utils makes sense to me